change/@fluentui-react-virtualizer-701a2cd4-bd8c-48af-a2d4-753903d62124.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33114&commitId=e1506254fed1d155d18560cb2a1f7641f8c7bdb2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|997| Changed|
|Drawer.overlay drawer full.chromium.png|3302| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>